tests/conftest.py

```diff
@@ -158,11 +158,8 @@ def __init__(self):
         print 'Gas Used: %s' % (self.chain.head_state.gas_used - startingGas)
         self.scalarMarket = self.createReasonableScalarMarket(self.universe, 40, self.cash)
         self.constants = self.uploadAndAddToController("solidity_test_helpers/Constants.sol")
-        self.chain.mine(1)
-        self.originalHead = self.chain.head_state
-        self.originalBlock = self.chain.block
-        self.originalContracts = deepcopy(self.contracts)
-        self.snapshot = self.chain.snapshot()
+        self.captured = {}
+        self.createSnapshot
```

Comment:
Style: Add `()` at the end of this line so it is clear that this is a function call.